clojure-mode.el

```diff
@@ -2270,6 +2270,16 @@ This will skip over sexps that don't represent objects, so that ^hints and
           (backward-sexp 1))
         (setq n (1- n))))))
 
+(defun clojure--is-top-level-form-p (&optional point)
```

Comment:
What's the difference with `clojure-top-level-form-p`? It's a bit confusing to have 2 very similarly named functions IMO.

Reply:
`clojure--is-top-level-form-p` checks if the sexp following the point is a top level form.
`clojure-top-level-form-p` checks the type of enclosing top level form w.r.t current point.
ex:
`|(ns foo)` -> `(clojure--is-top-level-form-p)` returns true.
`(|ns foo)` -> `(clojure--is-top-level-form-p)` returns nil.
`(comment |(ns foo))` -> `(clojure--is-top-level-form-p)` returns nil.
`(comment (|ns foo))` -> `(clojure-top-level-form-p "commennt")` returns true.

Reply:
I'd suggest naming it `clojure--looking-at-top-level-form`

This would be consistent in name and semantics with `looking-at`, I believe.

Reply:
Renamed the fn to `clojure--looking-at-top-level-form`.